src/vmm/src/memory_snapshot.rs

```diff
@@ -200,18 +207,23 @@ impl SnapshotMemory for GuestMemoryMmap {
     }
 }
 
+fn get_page_size() -> i64 {
+    unsafe { libc::sysconf(libc::_SC_PAGESIZE) }
+}
```

Comment:
I would include _error parsing_ in this function instead of moving that to the caller:
```suggestion
fn get_page_size() -> Result<usize> {
    match unsafe { libc::sysconf(libc::_SC_PAGESIZE) } {
        -1 => Err(Error::PageSize(errno::Error::last())),
        ps => Ok(ps),
    }
}
```

Reply:
Done